### ISSUE TITLE
Change access to production DBS to go through cmsweb-prod url

### ISSUE THREAD

**Issue by amaltaro** [dmwm/WMCore#10248]
**Impact of the new feature**
WMCore in general (and any clients depending on it)

**Is your feature request related to a problem? Please describe.**
As discussed in the DBS chat on the Jan 21st, we have decided to move all the "production" clients to a different cmsweb url / frontend. The standard "cmsweb" domain will be left for standard users and activity, while "cmsweb-prod" will be used by our COMP DMWM software to access DBS backends on the VM.

**Describe the solution you'd like**
All the default urls pointing to DBS `cmsweb.cern.ch` must be replaced by `cmsweb-prod.cern.ch`.
This must be done in every single possible location where a DBS url is formed, such as:
* configuration files
* source code
* test code
* deployment scripts
* services_config configurations
* anything else?

Again, note that this change is only required for DBS access, all the other services within CMSWEB shall remain pointing to cmsweb.cern.ch

**Describe alternatives you've considered**
none

**Additional context**
In short, this is what we agreed a couple of weeks ago:
```
Step 1
* Configure cmsweb-prod monitoring pages, CMSMONIT-328
* Move crab to use cmsweb-prod
Step 2
* Decommission cmsweb-k8s-prod
Step 3
* Move WMcore clients to cmsweb-prod
Step 4 
* Configure cmsweb on k8s FE and decommission from VM
Step 5
* Gradually moving DBS to K8s on cmsweb.cern.ch
```

@klannon Kevin, did I misunderstand any of those steps we agreed upon?

**Comment by amaltaro** [comment]
WMCore services running in kubernetes production now have all the necessary configuration changes in, as provided in these merge request:
https://gitlab.cern.ch/cmsweb-k8s/services_config/-/merge_requests/69

In order to cover 100% of the DBS calls, we need however to update the source code to translate workflow spec from cmsweb to cmsweb-prod, to happen only on 2nd of March...
